### PR TITLE
Update readme

### LINKED ISSUE #113: Update README file
Update the `README.md` file to reflect the latest updates on Akira

- Update the screenshot
- Remove Kickstarter campaign link
- Remove assets and redirect to dedicated repo
- Add LiberaPay link

### PR DESCRIPTION
**Pull request by Alecaddd**
Update the readme file with new layout, updated donation links, and better organized content.

- Closes #113

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,10 +43,25 @@ meson build --prefix=/usr
...
+## Questions and issues
+If you want to ask any question about the project, we have a dedicated Discord channel available to any [Patreon](https://www.patreon.com/akiraux) supporter.
+
+If you are trying out Akira and you encounter an error or any problem feel free to just open an issue.
+
+## Donations
+If you like Akira and you want to support its development, consider donating via [Liberpay](https://liberapay.com/AkiraUX/) or pledge on [Patreon](https://www.patreon.com/akiraux).
+
+### Official Mascot
+![](akira-mascot-akari.png)
```
Comment (bilelmoussaoui) [inline README.md]: We should use the image from the assets repository and remove it from the source repo.

**Comment by bilelmoussaoui** [comment]
We should put that screenshot under assets repo as it's not the current state of the application.

**Comment by sanecito** [comment]
I'd also recommend a slight addition to this to make donations more visible in the README; you should add a LiberaPay badge at the top of the README, to the right of the build badge similar to Lutris: https://github.com/lutris/lutris/blob/master/README.rst

File: README.md (hunk 1 above)
Comment (Alecaddd) [reply README.md]: Updated

**Pull request merged**